src/components/SecondaryNav/_secondary-nav.scss

```diff
@@ -1,14 +1,16 @@
+@import "vanilla-framework/scss/settings_colors";
```

Comment:
Can you confirm that sass dedupes these?

Reply:
Yes, confirmed.